docs/QuestionsIssuesPrs.md

```diff
@@ -38,7 +38,7 @@ If you don't get an answer within 1-2 days, feel free to post another comment in
 
 ### PR workflow
 
-1. **Select an issue to fix** 
+1. #### Select an issue to fix
```

Comment:
Be careful with using undocumented features though. It can be a bug that was allowed to live as a feature but liable to be killed later. :-p